sdk/search/Azure.ResourceManager.Search/api/Azure.ResourceManager.Search.netstandard2.0.cs

```diff
@@ -153,6 +156,56 @@ public partial class SharedSearchServicePrivateLinkResourceData : Azure.Resource
 }
 namespace Azure.ResourceManager.Search.Models
 {
+    public enum AadAuthFailureMode
+    {
+        Http403 = 0,
+        Http401WithBearerChallenge = 1,
+    }
+    public partial class DataPlaneAuthOptions
```

Comment:
Could we rename the name of this class to `SearchAadAuthDataPlaneAuthOptions` to specify?